test/fixtures/wsl_products.yml

```diff
@@ -0,0 +1,19 @@
+# TODO: references for this file,
+# describe what it is
+# https://jira.suse.com/browse/PED-1380
+- arch: "$arch"
+  display_name: "SUSE Linux Enterprise Desktop 15 SP4"
+  name: "SLED"
+  product_line: "sled"
+  register_target: "sle-15-$arch"
+  version: "15.4-0"
+  version_version: "15.4"
+- arch: "$arch"
+  display_name: "SUSE Linux Enterprise Server 15 SP4"
```

Comment:
FIXME: `default: true` is missing here

Reply:
BTW do we need that arch key here at all? maybe we can just always add current architecture, not?